### PR TITLE
[TW-1530] Refresh /docs/getting-started/first-steps/overview 

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Rewrite the featured snippet per guidelines
- Add details about how to use first steps
- Update date

### CODE REVIEW

File: src/pages/docs/getting-started/first-steps/overview.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ contextual_links:
     url:  "https://academy.postman.com/path/api-beginner"
 ---
 
-Welcome to Postman! This section of the Learning Center contains a quick guide on how to first get started with the Postman app.
+Follow this quick guide to get started using Postman. You'll learn how to install the Postman app, send an API request, write your first test, and save requests in a collection. You'll also learn about working with your team in Postman and discovering APIs on the Public API Network.
 
-In this section, you'll learn how to get the Postman app, send an API request, write your first test, and save requests to a Postman Collection. You'll also learn about working with your team in Postman and using the Public API Network.
+When you're ready to begin, select **Download** in the navigation at left. Postman recommends going through each of the topics in this section in order. Make sure to sign up for a free Postman account when directed so you can save your work.
```
Comment (petewitcher-postman) [inline src/pages/docs/getting-started/first-steps/overview.md:40]:
```suggestion
When you're ready to begin, select **Download** in the navigation menu at left. Postman recommends going through each of the topics in this section in order. Make sure to sign up for a free Postman account when directed so you can save your work.
```

**Review by petewitcher-postman — APPROVED** [review]
LGTM! (suggested "navigation menu")

**Review by nerminamiller-postman — APPROVED** [review]
Nice work, Mark!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**